x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformPersistentTasksExecutor.java

```diff
@@ -139,9 +139,13 @@ protected void nodeOperation(AllocatedPersistentTask task, @Nullable DataFrameTr
 
         final SetOnce<DataFrameTransformState> stateHolder = new SetOnce<>();
 
-        ActionListener<StartDataFrameTransformTaskAction.Response> startTaskListener = ActionListener.wrap(
-            response -> logger.info("Successfully completed and scheduled task in node operation"),
-            failure -> logger.error("Failed to start task ["+ transformId +"] in node operation", failure)
+        ActionListener<StartDataFrameTransformAction.Response> startTaskListener = ActionListener.wrap(
+            response -> logger.info("[{}] successfully completed and scheduled task in node operation", transformId),
+            failure -> {
+                auditor.error(transformId, "Failed to start data frame transform. " +
```

Comment:
I noticed while troubleshooting, that if we fail on `start` within the executor, the user could never be the wiser. We should at least audit here. 

I will experiment to see if setting the task to failed here is OK.

They will not be able to call `_start` again as the task already exists, it just won't be flagged as `FAILED`. 
Maybe we want to simply audit and call `.stop` on the task?

Additionally, this type of failure is present in 7.4. We may want to at least add an audit message there before release.

Reply:
@droberts195 @hendrikmuhs ^^^

Reply:
adding an audit for 7.4.1 sounds good to me.

As for "If anything goes wrong within `startTask`" - I think marking the task as failed and setting a reason is the preferable solution.